Python/View_and_Combine_STL/README.md

```diff
@@ -0,0 +1,24 @@
+# View and Combine STL
+
+STL files are an important part of 3D asset creation and management. STL files provide us with basic maps of 3D objects created/edited in applications like Blender/Unreal/Unity
+
+STL are mainly used to create 3D maps of environments like cityscapes and heightmaps of terrain. 
+
+## Running the Script
+* Create a virtual env
+* install numpy-stl, numpy and matplotlib or use requirements.txt
+* You can use combine.py to concatenate multiple STL files(recommended for assets designed in same 3D space or domain)
+* Or you can simply run view.py to view the STL file in a pyplot env.(set the target size to your liking)
```

Comment:
Please explain why. You can consider that I'm a beginner with no idea whatsoever, and want to use this. How am I going to use it?